src/swagger-specs/destination-authoring.yaml

```diff
@@ -0,0 +1,2975 @@
+          required: true
+          type: string
+        - name: "instance_id"
+          in: "path"
+          description: "The ID of the destination server configuration you want to delete."
+          required: true
+          type: "string"
+      responses:
+        "200":
+          description: "successful operation"
+          schema:
+            type: "object"
+            items:
+              $ref: "#/definitions/destinationServer"
+#      security:
+#       - dest_auth:
+#            - "write:dest"
+#            - "read:dest"
+  /audience-templates:
+    post:
+      tags:
+        - "Audience metadata templates"
+      summary: "Create a new audience template"
+      description: "You can create a new audience template by making a POST request to the `/authoring/audience-templates` endpoint."
+      operationId: "addAudienceTemplate"
+      consumes:
+        - "application/json"
+      produces:
+        - "application/json"
+      parameters:
+        - name: Authorization
+          in: header
+          description: "The access token which can be copied from your Experience Platform integration, prefixed with 'Bearer '. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+          required: true
+          type: string
+        - name: x-api-key
+          in: header
+          description: "The API key which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+          required: true
+          type: string
+        - name: x-gw-ims-org-id
+          in: header
+          description: "The organization ID which can be copied from your Experience Platform integration. For more information on how to obtain this value, visit the [authentication tutorial](http://www.adobe.com/go/platform-api-authentication-en)."
+          required: true
+          type: string
+        - name: x-sandbox-name
+          in: header
+          description: "All resources in Experience Platform are isolated to specific virtual sandboxes. Requests to Platform APIs require a header that specifies the name of the sandbox in which the operation will take place."
+          required: true
+          type: string
+        - in: "body"
+          name: "body"
+          description: "Audience template configuration object for a new destination. See the configuration examples in the [audience metadata functionality document](http://www.adobe.com/go/destination-sdk-audience-metadata-en)."
+          required: true
+          schema:
+            $ref: "#/definitions/audienceTemplate"
+      responses:
+        "405":
+          description: "Invalid input"
```

Comment:
More missing success responses. Please ensure that all operations have at least one successful response with a complete example.